### ISSUE TITLE
Deprecation CHEF-3694 in 4.0.1

### ISSUE THREAD

**Issue by marcmillien** [chef-cookbooks/lvm#114]
4.0.1 introduced a deprecation notice
```
Deprecated features used!
  Cloning resource attributes for chef_gem[di-ruby-lvm-attrib] from prior resource (CHEF-3694)
Previous chef_gem[di-ruby-lvm-attrib]: /tmp/kitchen/cache/cookbooks/lvm/libraries/lvm.rb:23:in `require_lvm_gems'
Current  chef_gem[di-ruby-lvm-attrib]: /tmp/kitchen/cache/cookbooks/lvm/libraries/lvm.rb:23:in `require_lvm_gems' at 1 location:
    - /tmp/kitchen/cache/cookbooks/lvm/libraries/lvm.rb:23:in `require_lvm_gems'
  Cloning resource attributes for chef_gem[di-ruby-lvm] from prior resource (CHEF-3694)
Previous chef_gem[di-ruby-lvm]: /tmp/kitchen/cache/cookbooks/lvm/libraries/lvm.rb:28:in `require_lvm_gems'
Current  chef_gem[di-ruby-lvm]: /tmp/kitchen/cache/cookbooks/lvm/libraries/lvm.rb:28:in `require_lvm_gems' at 1 location:
    - /tmp/kitchen/cache/cookbooks/lvm/libraries/lvm.rb:28:in `require_lvm_gems'
```

### Cookbook version
4.0.1

### Chef-client version
12.16

### Platform Details
Linux

### Scenario:
`chef-client` execution.

**Comment by tas50** [comment]
I can work around these, but they're actually harmless. In this scenario the full deprecation of the feature will have zero impact.

**Comment by marcmillien** [comment]
I confirm, it just a warning ^^. I can help, I just don't know how it should be done.

**Comment by tas50** [comment]
If you make sure the resources have a unique name you can avoid it. We can move the package name out of the resource name and instead make the package name also include the name  of the called LVM resource. Instead of having X number of accurences of 'package something' you have 'package some lvm_whatever' and they're all unique.

**Comment by cheeseplus** [comment]
Fixed by #117
